### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Fixes: [code quality ](https://github.com/ballerina-platform/ballerina-standard-library/issues/2310)
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Review by kaneeldias — APPROVED** [review]

**Comment by daneshk** [comment]
@kalaiyarasiganeshalingam shall we rename the bal file name to `scheduler_test.bal`. We normally don't use camel case for filenames

**Comment by kalaiyarasiganeshalingam** [comment]
> @kalaiyarasiganeshalingam shall we rename the bal file name to `scheduler_test.bal`. We normally don't use camel case for filenames

Renamed in 3c534a8

File: ballerina/tests/scheduler_test.bal [hunk 1]
```diff
@@ -514,10 +514,10 @@ class Job22 {
 function testIntervalJobWithEndTime() returns error? {
     time:Utc currentUtc = time:utcNow();
     time:Utc newTime = time:utcAddSeconds(currentUtc, 5);
-    time:Civil startTime = time:utcToCivil(currentUtc);
+    time:Civil _ = time:utcToCivil(currentUtc);
```
Comment (daneshk) [inline ballerina/tests/scheduler_test.bal:517]:
```suggestion
    _ = time:utcToCivil(currentUtc);
```

File: ballerina/tests/scheduler_test.bal [hunk 2]
```diff
@@ -485,9 +485,9 @@ function testIntervalJobWithStattTime() returns error? {
     time:Utc currentUtc = time:utcNow();
     time:Utc newTime = time:utcAddSeconds(currentUtc, 3);
     time:Civil startTime = time:utcToCivil(currentUtc);
-    time:Civil endTime = time:utcToCivil(newTime);
+    time:Civil _ = time:utcToCivil(newTime);
```
Comment (daneshk) [inline ballerina/tests/scheduler_test.bal:488]:
```suggestion
    _ = time:utcToCivil(newTime);
```

**Review by daneshk — APPROVED** [review]

**Pull request merged**